docs/api/README.md

```diff
@@ -1,34 +1,28 @@
 <p align="center">
-  <img src="https://github.com/energywebfoundation/iam-client-lib/actions/workflows/deploy.yml/badge.svg" />
```

Comment:
I guess this change will not take effect, because docs/api are removed on commit. Probably README is copied here, so there is enough to change it only in one palce